Add chaos resource debugging to readme (#2409)
Azure/azure-sdk-tools: tools/stress-cluster/chaos/README.md

### Faults via Dashboard

> NOTE: The chaos mesh dashboard is just a helper for generating manifest under the hood. You can create and submit these directly as well. See the [docs](https://chaos-mesh.org/docs/chaos_experiments/networkchaos_experiment).

Author: ```suggestion
NOTE: The chaos mesh dashboard is just a helper for generating manifest under the hood. You can create and submit these directly as well. See the [docs](https://chaos-mesh.org/docs/simulate-network-chaos-on-kubernetes/).
```

See [Chaos Manifest](#chaos-manifest).

### Debugging chaos resources and events

There are a few ways to check on the status of your chaos resources, after your stress test pod(s) reach a `Running` state.

> From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashbaord), select your stress test pods from the dropdown

Reviewer: ```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```
typo:
```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```

> From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashbaord), select your stress test pods from the dropdown

Reviewer: typo
```suggestion
From the [test cluster dashboard](https://aka.ms/azsdk/stress/dashboard), select your stress test pods from the dropdown
```
